core/src/main/java/nu/yona/server/goals/entities/TimeZoneGoal.java

```diff
@@ -25,8 +26,8 @@
 @Entity
 public class TimeZoneGoal extends Goal
 {
-	@ElementCollection
-	private List<String> zones;
+	@Column(length = 24 * 4 * 6)
```

Comment:
Why this length? I would expect that there are 24*4 quarters of an hour in a day, so that you can indeed create 24*4 non-overlapping zone specifications. However, every zone specification is 11 characters "hh:mm-hh:mm".